tests/e2e/pages/popup.ts

```diff
@@ -122,14 +122,29 @@ export type ConnectDetails = {
   recurring: boolean;
 };
 
-export async function fillPopup(popup: Popup, params: ConnectDetails) {
-  await popup
-    .getByLabel('Wallet address or payment pointer')
-    .fill(params.walletAddressUrl);
-  await popup.getByLabel('Amount', { exact: true }).fill(params.amount);
-  await popup
-    .getByLabel('Renew amount monthly')
-    .setChecked(params.recurring, { force: true });
+export async function fillPopup(popup: Popup, params: Partial<ConnectDetails>) {
```

Comment:
`params` are `Partial<ConnectDetails>` now as we disable other inputs unless a correct `walletAddress` URL is added. The validation is triggered on input blur. So, we now fill in parts.